game-core/src/main/java/org/triplea/game/server/AvailableGames.java

```diff
@@ -58,6 +49,16 @@
             availableGames.keySet().size(), availableGames.keySet()));
   }
 
+  private static Map<String, URI> getGames(final File map) {
+    log.info("Loading map: " + map);
+    if (map.isDirectory()) {
+      return getGamesFromDirectory(map);
+    } else if (map.isFile() && map.getName().toLowerCase().endsWith(ZIP_EXTENSION)) {
+      return getGamesFromZip(map);
+    }
+    return Map.of();
```

Comment:
side-note: I think this original error handling is extremely questionable. Since this is a bot server, we should not be in a no-op case, this indicates a likely configuration error. Returning a no-op is probably a silent error.